README.md

```diff
@@ -100,9 +128,26 @@ secret/kcp-syncer-kind-25coemaz created
 deployment.apps/kcp-syncer-kind-25coemaz created
 ```
 
+Now you need to use `kubectl kcp bind compute` to let `kcp` know about this new `kind` cluster that is providing compute resources. You do that by running: 
+
+```
+kubectl kcp bind compute "root:users:zu:yc:kcp-admin:locations"
+```
+
+Notice that we are pointing to our `locations` workspace that we created before. 
+
+Once you created the binding you can create a new workspace for deploying your applications. First you move to your home workspace and then you can create a `myapps` workspace by running:
+
+```
+kubectl kcp workspace
+kubectl kcp workspace create "myapps" --enter
```

Comment:
I don't think it's clear why you now need to care about workspaces for this "hello world" scenario. In the existing quickstart everything happens in the default workspace ("root" I guess). (I'm actually having trouble using the existing quickstart with main now, but it works for v0.9.1.)

In this new quickstart, two workspaces are created, and it's not really clear why they are needed. It's easy to think kcp -> pcluster, but the user needs more context to figure out myapps -> location -> pcluster.

Also, why "location" and why "myapps"? I know you don't want to complicate things on the main README, but I think it might be confusing as is.

Reply:
@merlante I was hitting issues with the readme and this was suggested by @davidfestal , and now the readme works out of the box. I think that the intention is to have a clear separation between where you are going to deploy workloads and other workspaces that are more for admin stuff. 
I was thinking about diagrams.. this getting started guide is really complex for new people and some diagrams might help a lot.

Reply:
Hi @salaboy, I hit issues with the readme as well with the main branch, and so I agree it needs to be changed. :) I understand (mostly) the reason for the two workspaces, but I'm concerned that an average user coming to the site will have no idea. What is there right now might have issues, but at least it's easy enough to grasp. With these changes, we're moving to something more correct, but that is not completely intuitive or easy to understand.

One option would be to maybe get this PR merged in it's current form -- assuming that the steps are fully working (I will check today if I get the chance) -- and then add a separate PR afterwards to add disagrams/context.

Reply:
@merlante I commit to add the diagram to explain what is going on in a separate PR, if you manage to check that the steps here are working, let me know. Notice that this PR also add the KIND cluster creation command, without that is was a bit too confusing for a newbie like me.

Reply:
I tend to agree with @merlante that for a Quickstart we should only document the minimum number of steps to allow users to kick the tires. On the other hand, it could be useful to explain scenarios where the synctarget and location are created in a user-created workspace and consumed from another user-created workspace (btw, I have been exercising that scenario as well using the `main` branch and run into this [issue](https://github.com/kcp-dev/kcp/issues/2737). I would suggest to leave the Quickstart as simple as possible, and add readme for the two workspaces scenarios or more advanced use cases under the "Next Steps" section.